README.md

```diff
@@ -137,57 +142,35 @@ Incompatible changes require incrementing the major version number. In our curre
 parts of the API without releasing a version `1.0.0`.
 
-## Deployment
-
-To push versions of this repository to different environments (including per-developer
-integration environments), update the
-[terra-helmfile deployment definitions](https://github.com/broadinstitute/terra-helmfile).
-
 ### On commit to main
+
 1. New commit is merged to main
-2. [The tag-publish workflow](https://github.com/DataBiosphere/terra-workspace-manager/blob/dev/.github/workflows/tag-publish.yml) is triggered. It builds the image, tags the image & commit, and pushes the image to GCR. It then sends a [dispatch](https://help.github.com/en/actions/reference/events-that-trigger-workflows#external-events-repository_dispatch) with the new version for the service to the [terra-helmfile repo](https://github.com/broadinstitute/terra-helmfile).
-3. This updates the default [version mapping for the app in question](https://github.com/broadinstitute/terra-helmfile/blob/master/versions.yaml).
-4. [Our deployment of ArgoCD](https://ap-argocd.dsp-devops.broadinstitute.org/applications) monitors the above repo, and any environments in which the app is set to auto-sync will immediately pick up the new version of the image. If the app is not set to auto-sync in an environment, it can be manually synced via the ArgoCD UI or API.
+2. [The tag-publish workflow](https://github.com/DataBiosphere/terra-workspace-manager/blob/dev/.github/workflows/tag-publish.yml)
+   is triggered. It builds the image, tags the image & commit, and pushes the image to GCR. It then sends
+   a [dispatch](https://help.github.com/en/actions/reference/events-that-trigger-workflows#external-events-repository_dispatch)
+   with the new version for the service to the [terra-helmfile repo](https://github.com/broadinstitute/terra-helmfile).
+3. This updates the
+   default [version mapping for the app in question](https://github.com/broadinstitute/terra-helmfile/blob/master/versions.yaml).
+4. [Our deployment of ArgoCD](https://ap-argocd.dsp-devops.broadinstitute.org/applications) monitors the above repo, and
+   any environments in which the app is set to auto-sync will immediately pick up the new version of the image. If the
+   app is not set to auto-sync in an environment, it can be manually synced via the ArgoCD UI or API.
 
 ## Release to Upper Environments (Staging, Prod)
 
 WSM follows standards for deployment via DSP DevOps services
 ([Beehive](https://beehive.dsp-devops.broadinstitute.org/)).
 
-For more information on how to release WSM to upper environments (including on demand via independent release), please see
+For more information on how to release WSM to upper environments (including on demand via independent release), please
+see
 [Releasing our Code](https://broadworkbench.atlassian.net/wiki/spaces/WOR/pages/2974449675/Releasing+our+Code)
 in Confluence.
 
 ## Setup
 
 ### Prerequisites:
 
-#### Postgres
-We are currently using Postgres 13.1.
```

Comment:
I don't think running WSM with a local postgres is supported anymore (I'm unaware of anyone running in this mode). I want to centralize on a single approach for turning on the DB.

Reply:
I don't think anyone really runs postgres outside of a docker container _anywhere_, anymore.